src/TwigComponent/src/EventListener/PreRenderEvent.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\EventListener;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class PreRenderEvent extends Event
+{
+    public function __construct(
+        public object $component,
+        public string $template,
+        public array $context,
```

Comment:
Maybe `$variables` instead? These are literally the Twig variables I believe

Reply:
I chose `$context` because it matches the `$twig->render()` variable name. Also, I noticed in Symfony, `TemplatedEmail`, uses `$context`. I don't have strong opinion on this though.